source/geod24/LocalRest.d

```diff
@@ -144,10 +131,10 @@ private struct TimeCommand
 }
 
 /// Ask the node to shut down
-private struct ShutdownCommand (API)
+private struct ShutdownCommand
 {
     /// Any callback to call before the Node's destructor is called
-    void function (API) callback;
+    void function (Object) callback;
```

Comment:
Why the type erasure ? We could just template `Variant`, couldn't we ?

Reply:
We would need to template `CommandChn`, `Connection` and `BindChn` too, right? I wanted to avoid that.